akka-cluster-typed/src/test/scala/akka/cluster/typed/ClusterSingletonPersistenceSpec.scala

```diff
@@ -33,7 +33,7 @@ object ClusterSingletonPersistenceSpec {
 
   val persistentActor: Behavior[Command] =
     EventSourcedBehavior[Command, String, String](
-      persistenceId = PersistenceId("TheSingleton"),
+      persistenceId = PersistenceId.ofUniqueId("TheSingleton"),
```

Comment:
That is one mouthful, could we just keep the `.of` and call the parameter `uniqueId` instead?

Reply:
`.of(` is already taken for the ordinary Java factory methods, so would have same problem as stated in the motivation of this change.

`ofUniqueId` is a power user api so will not be used by many

Reply:
Ah, missed that aspect. Then it's fine.